scipy/linalg/tests/test_lapack.py

```diff
@@ -1793,6 +1793,188 @@ def test_getc2_gesc2():
                                       x/scale, decimal=4)
 
 
+@pytest.mark.parametrize('size', [(6, 5), (5, 5)])
```

Comment:
The test still fails if shape is `(m, 1)` for some combinations of job? parameters. I'll have to do some digging to see why that occurs. Either way, rather than create more branches inside this test, I'll either add another test or add some combinations to ``specific`` tests.